api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -70,26 +86,26 @@
 
   @Override
   public Optional<Institution> getInstitution(final String shortName) {
-    return getDbInstitution(shortName)
+    return institutionDao
+        .findOneByShortName(shortName)
         .map(institution -> institutionMapper.dbToModel(institution, this));
   }
 
   @Override
   public DbInstitution getDbInstitutionOrThrow(final String shortName) {
-    return getDbInstitution(shortName)
+    return institutionDao
+        .findOneByShortName(shortName)
         .orElseThrow(
             () ->
                 new NotFoundException(String.format("Could not find Institution '%s'", shortName)));
   }
 
-  private Optional<DbInstitution> getDbInstitution(final String shortName) {
-    return institutionDao.findOneByShortName(shortName);
-  }
-
   @Override
   public Institution createInstitution(final Institution institutionToCreate) {
```

Comment:
Why are you declaring method parameters final? I'm a pretty big finalphile, but I don't generally do that.

In the case of calling `save()`, you should technically be nullling out its argument or replacing it so you don't accidentally reuse it. Though in practice it's less dangerous than the doc makes it sound.

Reply:
I don't have a specific reason to make them final.  I also don't have a specific reason not to - so my default is to make all things final unless I have a good reason.

The argument to save here is never stored in a variable so there's nothing to null out

Reply:
Oh, I misread it. The thing being saved is anonymous, so you're good.

My default is to make everything final that could possibly be final, with the exception of method parameters, and that's only for readability.